conf/jvm11-server.options

```diff
@@ -4,6 +4,14 @@
 # See jvm-server.options. This file is specific for Java 11 and newer.    #
 ###########################################################################
 
+
+########################
+# GENERAL JVM SETTINGS #
+########################
+
+# Disable biased locking as it does not benefit Cassandra.
```

Comment:
You are changing options for JDK people already use in a certain way. I would refrain from that except if it is a bug fix. This is definitely not related also to what we do in this PR.